private_acl_rules.tf

```diff
@@ -312,6 +344,23 @@ resource "aws_network_acl_rule" "private_egress_to_cool_via_ephemeral_ports" {
   to_port        = 65535
 }
 
+# Allow egress to COOL Shared Services via UDP ephemeral ports
+#
+# For: Assessment team access to Advanced Operations VPN endpoints
```

Comment:
If I'm not mistaken, this is the bit that was confusing us earlier this week.  Did we ever find any documentation that really explained why this is necessary?  Can we include some explanatory text and/or a link here?

Reply:
I think you might be mixing this up with the changes I made in https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/a3f8be86353918eaf593eeda6a3b45223c08c3cb and then just corrected in https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/34051abde3b014b5e82eda1945769a0d7edabbb3.  This rule is necessary to allow return traffic on UDP ephemeral ports to flow from the transit gateway attachment to the Shared Services subnet (e.g. VPN-connected laptops used by assessors).  I think I could make this rule more restrictive so that it only allows this traffic out from the first private subnet (where the TGW attachment lives), but since we already [allow TCP ephemeral ports back to COOL Shared Services from both private subnets](https://github.com/cisagov/cool-assessment-terraform/blob/develop/private_acl_rules.tf#L298-L313), I figured it was fine to do the same for UDP.

Reply:
OK, I got it.